tests/integration/client/feedback/training/test_openai.py

```diff
@@ -0,0 +1,82 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import re
+from typing import TYPE_CHECKING
+
+import argilla as rg
+import pytest
+from argilla.client.feedback.training.schemas import TrainingTaskForChatCompletionFormat
+
+if TYPE_CHECKING:
+    pass
```

Comment:
We should keep a line-break between the LICENSE header and the imports, `TYPE_CHECKING` is not needed here, and we shouldn't use `import argilla as rg` in the unit/integration tests, as that's a shortcut from the user-side, internally we should point to the actual modules instead

```suggestion

import re

import pytest
from argilla.client.feedback.dataset import FeedbackDataset
from argilla.client.feedback.training.schemas import TrainingTaskForChatCompletionFormat
```